utils/cluster_manager.py

```diff
@@ -341,10 +341,22 @@ def get_server_command() -> str:
             except Exception as e:
                 logging.error(f"Error checking {server}: {e}")
         raise Exception(
-            "Neither valkey-server nor redis-server found in the system."
+            "Neither valkey-server nor redis-server found in the system.")
```

Comment:
I used python client linters to groom this file

Reply:
actually this file has lots of changes bc of black but I tried to add only whats relevant